tests/scale/virt-api.go

```diff
@@ -25,29 +29,21 @@ var _ = Describe("[sig-compute] virt-api scaling", decorators.SigCompute, func()
 	numberOfNodes := 0
 
 	setccs := func(ccs v12.CustomizeComponents) (oldcss v12.CustomizeComponents) {
-		originalKv := util.GetCurrentKv(virtClient)
-		kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(context.Background(), originalKv.Name, v1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
+		kv := util.GetCurrentKv(virtClient)
 		oldcss = kv.Spec.CustomizeComponents
-		kv.Spec.CustomizeComponents = ccs
-		EventuallyWithOffset(1, func() error {
-			_, err = virtClient.KubeVirt(originalKv.Namespace).Update(context.Background(), kv, v1.UpdateOptions{})
-			return err
-		}, 30*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
-
+		patchPayload, err := patch.New(patch.WithReplace("/spec/customizeComponents", ccs)).GeneratePayload()
+		Expect(err).ToNot(HaveOccurred())
+		_, err = virtClient.KubeVirt(kv.Namespace).Patch(context.Background(), kv.GetName(), types.JSONPatchType, patchPayload, v1.PatchOptions{})
+		Expect(err).ToNot(HaveOccurred())
 		return oldcss
 	}
 
 	restorescc := func(ccs v12.CustomizeComponents) {
```

Comment:
`restorescc` and `setccs` seem very similar; `setcss` does two things (settinga new css and returning the old one).
Can you add a commit that merges them together?

Reply:
both `DeferCleanup(restorescc, oldcss)` in line 115 and restorescc func can be removed since the tests framework does the cleanup automatically

Reply:
I ended removing those methods since they were only used once. and the `restorescc` was redundant anyway.

Reply:
After discussing it with @dankenigsberg offline, we found out that the restore for the KubeVirt object happens [only for the configuration](https://github.com/kubevirt/kubevirt/blob/8016033c66e08163493ebdabd80bceb5d8329800/tests/tests_suite_test.go#L163). Therefore, we do need to restore it as it is not part of the KubeVirt configuration. 
sorry for the confusion

Reply:
Thanks for the clarification, it's fixed now.